gdk/common/config/TestConfiguration.py

```diff
@@ -14,6 +20,14 @@ def _set_build_config(self, test_build_config):
         self.test_build_system = test_build_config.get("build_system", self.test_build_system)
 
     def _set_gtf_config(self, test_config):
+        github_utils = GithubUtils()
+        try:
+            latest_gtf_version = github_utils.get_latest_release_name(GTF_REPO_OWNER, GTF_REPO_NAME)
+            self.gtf_version = latest_gtf_version
```

Comment:
We should make this API call only if the customer hasn't set their GTF version. For eg, if customer sets the `gtf_version` to 1.3.0 in the config, we don't need to make this call. We could verify if that version exists and warn them if it doesn't.

Reply:
In the future we will likely want to add an informational log line that notifies the user of GTF updates if they were to provide a version that isn't latest. To prepare for that addition, I'm running the API call every time.